src/Xablu.WebApiClient/Services/GraphQL/MutationRequest.cs

```diff
@@ -96,31 +93,22 @@ protected virtual void LoadProperties(Type type)
             }
         }
 
-        protected virtual void CurateProperties()
+        private string CreateQueryFromProperties()
         {
-            var parentTypeToExclude = typeof(List<>);
-            foreach (List<PropertyDetail> propertyList in _propertyListList.Where(list => list.Any()))
-            {
-                propertyList.RemoveAll(p => p.ParentName == parentTypeToExclude.Name);
-            }
-        }
-
-        private string QueryBuilder()
-        {
-            var variableInputTypeName = (Attribute.GetCustomAttribute(typeof(T), typeof(VariableInputTypeAttribute)) as VariableInputTypeAttribute)?.ModelInputName;
+            var variableInputTypeName = (Attribute.GetCustomAttribute(_requestObjectType, typeof(VariableInputTypeAttribute)) as VariableInputTypeAttribute)?.ModelInputName;
```

Comment:
When testing the new solution I noticed that the ConsoleApp does not work. The VariableInputAttribute that is mandatory can't be found. 

<img width="585" alt="Screenshot 2020-03-27 at 15 04 55" src="https://user-images.githubusercontent.com/22239299/77764255-a95cc780-703c-11ea-94d6-53028f38ac48.png">

Reply:
This is because it checks the ChangeUserInputVariables that has no attribute. ChangeUserStatus however does that's why typeOf(TResponseModel) was used.